.github/workflows/scripts/check-help-wanted.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+set -e
```

Comment:
I'd rather go with:

```sh
set -euxo pipefail
```

We can remove `x` when we're confident the script is doing okay.